### PR TITLE
new icon: vscodium (original, plain)

### LINKED ISSUE #2009: [ICON REQUEST]: vscodium
### I have searched through the issues and didn't find my problem.

- [X] Confirm

### About the icon

VSCodium is a telemetry-free binary release of VSCode as an alternative to the official distribution of Visual Studio Code, which is partially closed-source

### Links and sources

https://vscodium.com/

Icons dir:
https://github.com/VSCodium/vscodium/tree/master/icons

svg

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by femsci**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes #2009**

## Link to prove your SVG is correct and up-to-date.
codium

### CODE REVIEW

File: icons/vscodium/vscodium-original.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...81 127.988281 88.480469 127.996094 84.152344 C 128.003906 79.820312 124.492188 76.304688 120.148438 76.296875 C 118.082031 76.296875 116.066406 76.367188 114.082031 76.484375 C 117.460938 68.535156 118.960938 59.785156 118.648438 50.335938 C 118.507812 46.007812 114.875 42.613281 110.535156 42.753906 C 106.191406 42.898438 102.789062 46.523438 102.933594 50.851562 C 103.34375 63.226562 102.878906 74.277344 93.660156 80.105469 C 91.039062 81.757812 87.988281 83.195312 85.097656 83.195312 C 87.34375 77.09375 89.039062 70.648438 89.730469 63.707031 C 90.175781 59.273438 90.222656 54.007812 89.714844 49.921875 C 88.933594 43.585938 87.984375 36.398438 90.390625 30.984375 C 92.550781 26.324219 97.390625 24.371094 104.5 24.371094 C 108.835938 24.367188 112.351562 20.859375 112.351562 16.535156 C 112.355469 12.203125 108.839844 8.691406 104.5 8.6875 C 93.9375 8.6875 85.929688 14.246094 81.390625 20.960938 C 79.019531 15.894531 76.128906 10.671875 72.636719 5.261719 C 71.507812 3.515625 69.730469 2.289062 67.695312 1.847656 C 66.6875 1.628906 65.644531 1.609375 64.628906 1.789062 C 63.613281 1.972656 62.640625 2.355469 61.773438 2.910156 Z M 61.773438 2.910156" fill="url(#g0)" /></g></svg>
```
Comment (weh) [inline icons/vscodium/vscodium-original.svg:1]: The Image can be optimized further using https://www.svgviewer.dev/ (-40%)
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><defs><linearGradient id="a" gradientUnits="userSpaceOnUse" x1="199.997" y1="214.302" x2="296.665" y2="307.573" gradientTransform="matrix(1.28 0 0 1.28 -256 -272.64)"><stop offset="0" stop-color="#62A0EA"/><stop offset="1" stop-color="#1A5FB4"/></linearGradient></defs><path d="M61.773 2.91c-3.652 2.344-4.707 7.2-2.351 10.84C72.359 33.79 75.414 48.902 74.09 62.148c-5.356 25.364-16.996 30.508-27.172 30.508-9.934 0-7.273-16.344.195-21.129 4.457-2.785 10.164-4.593 14.547-4.593 4.34 0 7.86-3.508 7.86-7.836 0-4.332-3.52-7.84-7.86-7.84-5.117 0-10.14 1.078-14.824 2.906.957-4.516 1.309-9.398.07-14.586-1.879-7.883-7.285-15.406-16.68-22.703a7.856 7.856 0 0 0-5.8-1.598 7.855 7.855 0 0 0-5.235 2.961 7.826 7.826 0 0 0 1.38 11c7.652 5.946 10.179 10.344 11.042 13.965.867 3.621.172 7.504-1.566 12.844-2.219 7.191-4.793 13.613-5.922 19.773-.555 3.032-.602 6.328-.746 8.965-5.512-5.37-7.668-12.46-7.668-22.824 0-4.328-3.52-7.84-7.86-7.836-4.34.004-7.855 3.508-7.855 7.836 0 14.152 4.137 27.617 15.23 36.602 10.04 9.562 35.57 6.027 35.57 21.386 0 4.332 6.345 6.442 10.684 6.442 4.454 0 10.055-2.977 10.055-6.442 0-17.426 18.363-28.015 48.59-27.969 4.344.008 7.863-3.5 7.871-7.828.008-4.332-3.504-7.847-7.848-7.855-2.066 0-4.082.07-6.066.187 3.379-7.949 4.879-16.699 4.566-26.148-.14-4.328-3.773-7.723-8.113-7.582-4.344.144-7.746 3.77-7.601 8.098.41 12.375-.055 23.425-9.274 29.253-2.62 1.653-5.672 3.09-8.562 3.09 2.246-6.101 3.941-12.547 4.632-19.488.446-4.434.493-9.7-.015-13.785-.781-6.336-1.73-13.524.676-18.938 2.16-4.66 7-6.613 14.109-6.613 4.336-.004 7.852-3.512 7.852-7.836.003-4.332-3.512-7.844-7.852-7.848-10.563 0-18.57 5.56-23.11 12.274-2.37-5.066-5.261-10.29-8.753-15.7a7.867 7.867 0 0 0-4.942-3.413 7.943 7.943 0 0 0-3.066-.059 7.924 7.924 0 0 0-2.856 1.121Zm0 0" fill="url(#a)"/></svg>
```

File: icons/vscodium/vscodium-plain.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...281 127.988281 88.480469 127.996094 84.152344 C 128.003906 79.820312 124.492188 76.304688 120.148438 76.296875 C 118.082031 76.296875 116.066406 76.367188 114.082031 76.484375 C 117.460938 68.535156 118.960938 59.785156 118.648438 50.335938 C 118.507812 46.007812 114.875 42.613281 110.535156 42.753906 C 106.191406 42.898438 102.789062 46.523438 102.933594 50.851562 C 103.34375 63.226562 102.878906 74.277344 93.660156 80.105469 C 91.039062 81.757812 87.988281 83.195312 85.097656 83.195312 C 87.34375 77.09375 89.039062 70.648438 89.730469 63.707031 C 90.175781 59.273438 90.222656 54.007812 89.714844 49.921875 C 88.933594 43.585938 87.984375 36.398438 90.390625 30.984375 C 92.550781 26.324219 97.390625 24.371094 104.5 24.371094 C 108.835938 24.367188 112.351562 20.859375 112.351562 16.535156 C 112.355469 12.203125 108.839844 8.691406 104.5 8.6875 C 93.9375 8.6875 85.929688 14.246094 81.390625 20.960938 C 79.019531 15.894531 76.128906 10.671875 72.636719 5.261719 C 71.507812 3.515625 69.730469 2.289062 67.695312 1.847656 C 66.6875 1.628906 65.644531 1.609375 64.628906 1.789062 C 63.613281 1.972656 62.640625 2.355469 61.773438 2.910156 Z M 61.773438 2.910156" fill="#000000" /></g></svg>
```
Comment (weh) [inline icons/vscodium/vscodium-plain.svg:1]: same here (-44%)
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path d="M61.773 2.91c-3.652 2.344-4.707 7.2-2.351 10.84C72.359 33.79 75.414 48.902 74.09 62.148c-5.356 25.364-16.996 30.508-27.172 30.508-9.934 0-7.273-16.344.195-21.129 4.457-2.785 10.164-4.593 14.547-4.593 4.34 0 7.86-3.508 7.86-7.836 0-4.332-3.52-7.84-7.86-7.84-5.117 0-10.14 1.078-14.824 2.906.957-4.516 1.309-9.398.07-14.586-1.879-7.883-7.285-15.406-16.68-22.703a7.856 7.856 0 0 0-5.8-1.598 7.855 7.855 0 0 0-5.235 2.961 7.826 7.826 0 0 0 1.38 11c7.652 5.946 10.179 10.344 11.042 13.965.867 3.621.172 7.504-1.566 12.844-2.219 7.191-4.793 13.613-5.922 19.773-.555 3.032-.602 6.328-.746 8.965-5.512-5.37-7.668-12.46-7.668-22.824 0-4.328-3.52-7.84-7.86-7.836-4.34.004-7.855 3.508-7.855 7.836 0 14.152 4.137 27.617 15.23 36.602 10.04 9.562 35.57 6.027 35.57 21.386 0 4.332 6.345 6.442 10.684 6.442 4.454 0 10.055-2.977 10.055-6.442 0-17.426 18.363-28.015 48.59-27.969 4.344.008 7.863-3.5 7.871-7.828.008-4.332-3.504-7.847-7.848-7.855-2.066 0-4.082.07-6.066.187 3.379-7.949 4.879-16.699 4.566-26.148-.14-4.328-3.773-7.723-8.113-7.582-4.344.144-7.746 3.77-7.601 8.098.41 12.375-.055 23.425-9.274 29.253-2.62 1.653-5.672 3.09-8.562 3.09 2.246-6.101 3.941-12.547 4.632-19.488.446-4.434.493-9.7-.015-13.785-.781-6.336-1.73-13.524.676-18.938 2.16-4.66 7-6.613 14.109-6.613 4.336-.004 7.852-3.512 7.852-7.836.003-4.332-3.512-7.844-7.852-7.848-10.563 0-18.57 5.56-23.11 12.274-2.37-5.066-5.261-10.29-8.753-15.7a7.867 7.867 0 0 0-4.942-3.413 7.943 7.943 0 0 0-3.066-.059 7.924 7.924 0 0 0-2.856 1.121Zm0 0"/></svg>
```

**Review by weh — CHANGES_REQUESTED** [review]
Hi femsci, and welcome to Devicon :)

I could optimize the images for size using, svgviewer.dev, but very nice contribution :+1:

File: icons/vscodium/vscodium-plain.svg (hunk 2 above)
Comment (femsci) [reply icons/vscodium/vscodium-plain.svg:1]: I notice that both optimizations seem not to affect the rendering. I will accept them.

**Review by weh — APPROVED** [review]

**Review by canaleal — APPROVED** [review]
Excellent work 👍 
I added the missing fill color to the `plain svg`, but side from that, both icons look good. 

image

**Review by lunatic-fox — APPROVED** [review]
Great work! ✔️
Thank you for this contribution! 🚀

**Pull request merged**